Bricks/util/iterator.h

```diff
@@ -32,12 +32,12 @@ namespace current {
 
 template <typename MAP>
 struct GenericMapIterator final {
-  using iterator_t = typename MAP::const_iterator;
+  using inner_iterator_t = typename MAP::const_iterator;
```

Comment:
True indeed. Let's leave renaming to the last pass. Added a `DIMA_FIXME` for now to grep for it later, won't merge until we clean those up.